src/cdk/testing/protractor/protractor-element.ts

```diff
@@ -80,11 +80,11 @@ export class ProtractorElement implements TestElement {
     return this.element.clear();
   }
 
-  async click(...args: number[]): Promise<void> {
+  async click(...args: ['center'] | [number, number]): Promise<void> {
```

Comment:
Since we're on TS 4.0 now, you should be able to label the `[number, number]`, e.g. `[x: number, y: number]`.

Reply:
neat!

Reply:
Google is still on 3.9

Also, why `['center'] | [number, number]` and not `'center' | [number, number]`?